packages/api/src/makeMergedSchema/rootSchema.ts

```diff
@@ -53,7 +53,7 @@ export const resolvers: Resolvers = {
   Query: {
     redwood: () => ({
       version: apiPackageJson.version,
-      prismaVersion: PrismaClient.prismaVersion.client,
+      prismaVersion: PrismaClient.prismaVersion,
```

Comment:
Hmmmm... Maybe it's best to make this a resolver, can you wrap this in a function?
```suggestion
      prismaVersion: () => PrismaClient.prismaVersion,
```